api/tests/validate_legacy_soap_api/xsd/AgencyManagePackage-V1.0.xsd

```diff
@@ -0,0 +1,169 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xmime="http://www.w3.org/2005/05/xmlmime" xmlns:AgencyManagePackage="http://apply.grants.gov/system/AgencyManagePackage-V1.0" xmlns:GrantsCommonTypes="http://apply.grants.gov/system/GrantsCommonTypes-V1.0" xmlns:GrantsCommonElements="http://apply.grants.gov/system/GrantsCommonElements-V1.0" targetNamespace="http://apply.grants.gov/system/AgencyManagePackage-V1.0" elementFormDefault="qualified" attributeFormDefault="qualified" version="1.0">
```

Comment:
Could we make them be fetched by the script (and/or make a script to fetch and cache them)? That way we don't need to copy them all into our codebase / if things change we have the updates.

It's fine if it's a script that we have to hardcode the filepaths.

Reply:
I think I got this running by fetching and caching the xsds.